docs/modules/ROOT/pages/cpp-client-getting-started.adoc

```diff
@@ -0,0 +1,203 @@
+
+We will use the 5.1 version of Hazelcast for this tutorial.
+
+In this tutorial, we will use Docker for simplicity. You can also use https://docs.hazelcast.com/hazelcast/5.1/getting-started/get-started-cli[CLI], https://docs.hazelcast.com/hazelcast/5.1/getting-started/get-started-binary[Binary] and https://docs.hazelcast.com/hazelcast/5.1/getting-started/get-started-java[Maven].
+
+[source,bash]
+----
+docker run -p 5701:5701 hazelcast/hazelcast:5.1
+----
+
+This will start a new Hazelcast member at port 5701. Now, we have a Hazelcast cluster with just one member.
+
+== Install Hazelcast C++ Client
+In this tutorial we will use Vcpkg for installing the C++ client. You can also use https://github.com/hazelcast/hazelcast-cpp-client/blob/master/Reference_Manual.md#111-conan-users[Conan] or install from source using CMake as explained https://github.com/hazelcast/hazelcast-cpp-client/blob/master/Reference_Manual.md#113-install-from-source-code-using-cmake[here].
+
+Before starting download and install Vcpkg itself if you haven't already:
+[source,bash]
+----
+git clone https://github.com/microsoft/vcpkg
+.\vcpkg\bootstrap-vcpkg.bat
+----
+
+First, execute the following to install `hazelcast-cpp-client` ith its `boost` dependencies:
+
+[source,bash]
+----
+.\vcpkg\vcpkg install hazelcast-cpp-client
+----
+After the installation, the library is available for usage.
+For example, if you are using CMake for your builds, you can use the following cmake build command with the `CMAKE_TOOLCHAIN_FILE` cmake option to be the `vcpkg.cmake`.
+
+[source,bash]
+----
+cmake -B [build directory] -S . -DCMAKE_TOOLCHAIN_FILE=[path to vcpkg]/scripts/buildsystems/vcpkg.cmake
+cmake --build [build directory]
+----
+For more information about Vcpkg installation check https://github.com/hazelcast/hazelcast-cpp-client/blob/master/Reference_Manual.md#112-vcpkg-users[here].
+
+== Starting the C++ Client
+In this tutorial we use CMake for compilation, for other options you can check https://github.com/hazelcast/hazelcast-cpp-client/blob/master/Reference_Manual.md#13-compiling-your-project[here].
+If you are using CMake like we do, you can easily find and link against the client library:
+[source]
+----
+find_package(hazelcast-cpp-client CONFIG REQUIRED)
+
+target_link_libraries(mytarget PRIVATE hazelcast-cpp-client::hazelcast-cpp-client)
+----
+Make sure you add the installation prefix of the client library to CMAKE_PREFIX_PATH if you are using a custom installation location.
+
+Then, You can include the library and start a client using the following code:
+[source,cpp]
+----
+#include <hazelcast/client/hazelcast_client.h>
+
+int main() {
+    auto hz = hazelcast::new_client().get();
+}
+----
+
+The following line creates and starts a new Hazelcast C++ client with the default configuration.
+
+[source,cpp]
+----
+auto hz = hazelcast::new_client().get();
+----
+
+The client automatically connects to the Hazelcast member available on the local machine.
+
```

Comment:
The sample also disconnects the client upon destruction of the client? How does the other code samples do it?